### PR TITLE
Add tablet configuration section

### PR DESCRIPTION
**Pull request by peppy**
There's quite a bit here code-wise, but it's very contained so I've kept it in a single PR.

### CODE REVIEW

File: osu.Game/IO/Serialization/IJsonSerializable.cs [hunk 1]
```diff
@@ -28,7 +27,6 @@ public static class JsonSerializableExtensions
             Formatting = Formatting.Indented,
             ObjectCreationHandling = ObjectCreationHandling.Replace,
             DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate,
-            Converters = new JsonConverter[] { new Vector2Converter() },
```
Comment (smoogipoo) [inline osu.Game/IO/Serialization/IJsonSerializable.cs:31]: Are you sure this isn't required? This entire code is overriding the default settings after all.

File: osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+        private readonly BindableNumber<float> sizeX = new BindableNumber<float> { MinValue = 10 };
+        private readonly BindableNumber<float> sizeY = new BindableNumber<float> { MinValue = 10 };
+
+        [Resolved]
+        private GameHost host { get; set; }
+
+        /// <summary>
+        /// Based on the longest available smartphone.
+        /// </summary>
+        private const float largest_feasible_aspect_ratio = 20f / 9;
```
Comment (smoogipoo) [inline osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs:36]: Ultrawide PC configurations are commonly 21:9, so I'd like at least support up to that.

```suggestion
        private const float largest_feasible_aspect_ratio = 21f / 9;
```

**Review by smoogipoo — CHANGES_REQUESTED** [review]
Besides these comments, I've gone through the aspect stuff quite lightly and relied mostly on real-world testing to save time. Code looks sane.

File: osu.Game/IO/Serialization/IJsonSerializable.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/IO/Serialization/IJsonSerializable.cs:31]: I'll test, but presumed that the default set would still be used in this case.

File: osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs (hunk 2 above)
Comment (peppy) [reply osu.Game/Overlays/Settings/Sections/Input/TabletSettings.cs:36]: Fair enough. Comment needs to be updated too in that case.

File: osu.Game/IO/Serialization/IJsonSerializable.cs (hunk 1 above)
Comment (peppy) [reply osu.Game/IO/Serialization/IJsonSerializable.cs:31]: You're right, doesn't work that way. Will restore.

**Comment by smoogipoo** [comment]
The aspect ratio slider doesn't get an initial value set:

image

**Comment by peppy** [comment]
Note that even when disabling the handler, the settings will still show. I think this is for the best, to allow users to potentially disable while configuring the settings to avoid having their tablet jump around while doing so.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**